.github/workflows/test.yml

```diff
@@ -20,6 +20,7 @@ jobs:
       py36: 3.6
       py37: 3.7
       py38: 3.8
+      py39: 3.9
```

Comment:
This version will need to be added to the list on line 29 as well in order to be executed

Reply:
thanks for the pointer. updated